iosApp/RobotConf/UI/Agenda/Feedback/TalkFeedbackView.swift

```diff
@@ -25,7 +26,7 @@ struct TalkFeedbackView: View {
 
         switch content.availability {
         case .notAvailable:
-            return AnyView(Text(L10n.Agenda.Detail.Feedback.notAvailable).italic())
+                return AnyView(Text(stringResource(MR.strings().feedback_not_available)).italic())
```

Comment:
Don't want to bikeshed this too much but we might want to tweak the format options there. I'm guessing you're using Xcode to edit swift? Is there an option that controls this? Or can Xcode wotk with `.editorconfig`?

Reply:
I'll have a look at this. I think I'm gonna merge this and add an issue for codestyling iOS if you agree.

Reply:
Of course! Sounds like a plan 👍